care.js

```diff
@@ -311,12 +311,19 @@ function getAnsiArt(textToSay) {
     return ansiArt.get({ filePath: config.say, speechText: textToSay });
   }
 
+  var ansiOptions = {
+    artName: config.say,
+    speechText: textToSay,
+    speechBubbleOptions: { boxWidth: 50, boxType: 'classic' }
```

Comment:
Maybe replace the fixed `50` with `parrotBox.width-3` !? That would use the full width of the box.

Reply:
@mojoaxel Maybe use both? 
If parrotBox is less than 70 make speech bubble to 100% otherwise 70%??

👉 https://github.com/notwaldorf/tiny-care-terminal/pull/145/commits/4449689967bbf5f0f7edc121a3cd3eef6849855a